types/mongoose/index.d.ts

```diff
@@ -371,6 +372,8 @@ declare module "mongoose" {
     useNewUrlParser?: boolean;
     /** Set to false to make findOneAndUpdate() and findOneAndRemove() use native findOneAndUpdate() rather than findAndModify(). */
     useFindAndModify?: boolean;
+    /** Enables the new unified topology layer */
+    useUnifiedTopology?: boolean;
```

Comment:
See `mongodb.MongoClientOptions` in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/927dcd0af0ab47e790917fe9cc1860afc06640e1/types/mongodb/index.d.ts#L322 and http://mongodb.github.io/node-mongodb-native/3.4/api/MongoClient.html#.connect.